public/html/projects/templates/add.pug

```diff
@@ -20,10 +20,16 @@
 				input.form-control(type="text" placeholder="play.yml" ng-model="tpl.playbook")
 
 		.form-group
-			label.control-label.col-sm-4 SSH Key*
-			.col-sm-6
-				select.form-control(ng-model="tpl.ssh_key_id" ng-options="key.id as key.name disable when key.removed for key in keys")
-					option(value="") -- Select SSH Key --
+			label.control-label.col-sm-4 SSH key
+			.col-sm-6.col-sm-offset-4
+				.radio: label
+					input(type="radio" ng-model="tpl.user_key" name="user_key" data-ng-value="false")
+					select.form-control(ng-model="tpl.ssh_key_id" ng-options="key.id as key.name disable when key.removed for key in keys" ng-disabled="tpl.user_key==true")
+						option(value="") -- Select SSH Key --
+				.radio: label
+					input(type="radio" ng-model="tpl.user_key" name="user_key" data-ng-value="true")
+					| User specific SSH Key
```

Comment:
I'm not sure this is the right way to do this, because I can create more than 1 ssh key that belongs to me in a project, as they do not appear in the dropdown above I would expect that they get listed some other way, OR that it would be impossible for me to create more that 1 personal ssh key per project (and this was clearly explained by the dropdown where you pick the user who owns the key)